src/backend/api/Fusion.Resources.Domain/Commands/Contracts/UpdateContractExternalReps.cs

```diff
@@ -56,19 +53,33 @@ protected override async Task Handle(UpdateContractExternalReps request, Cancell
                     throw new InvalidOperationException($"Error trying to get error from org service. Received {ex.Response.StatusCode}, {ex.Error?.Message} ({ex.Error?.ErrorCode})", ex);
                 }
 
+                bool notifyCompanyRep = false;
+                bool notifyContractRep = false;
 
                 if (request.CompanyRepPositionId.HasBeenSet)
+                {
+                    if (request.CompanyRepPositionId.Value.HasValue && contract.ExternalCompanyRep?.Id != request.CompanyRepPositionId.Value.Value)
+                        notifyCompanyRep = true;
+
                     contract.ExternalCompanyRep = request.CompanyRepPositionId.Value.HasValue ? new ApiPositionV2 { Id = request.CompanyRepPositionId.Value.Value } : null;
+                }
 
                 if (request.ContractResponsiblePositionId.HasBeenSet)
+                {
+                    if (request.ContractResponsiblePositionId.Value.HasValue && contract.ExternalContractRep?.Id != request.ContractResponsiblePositionId.Value.Value)
+                        notifyContractRep = true;
+
                     contract.ExternalContractRep = request.ContractResponsiblePositionId.Value.HasValue ? new ApiPositionV2 { Id = request.ContractResponsiblePositionId.Value.Value } : null;
+                }
 
+                await orgClient.UpdateContractV2Async(request.OrgProjectId, contract);
```

Comment:
What if update fails? (Don't know, just asking for a friend)...

Reply:
That is unrelated to this PR I guess, but fair point still. We can add a Work item to improve this.